identity/app/controllers/EmailVerificationController.scala

```diff
@@ -48,7 +48,7 @@ class EmailVerificationController(api: IdApiClient,
           val verifiedReturnUrl = verifiedReturnUrlAsOpt.getOrElse(returnUrlVerifier.defaultReturnUrl)
           val encodedReturnUrl = URLEncoder.encode(verifiedReturnUrl, "utf-8")
 
-          if(validationState.isExpired || IdentityPointToConsentJourneyPage.isSwitchedOff) {
+          if(validationState.isExpired || IdentityRedirectUsersWithLingeringV1ConsentsSwitch.isSwitchedOff) {
```

Comment:
I don't think this switch name is appropriate any more - I preferred the other one that it was merged with.

Reply:
@jfsoul changed!